src/pages/nonMbLookup/NonMbLookup.tsx

```diff
@@ -8,78 +8,89 @@ import InputWrapper from "./components/InputWrapper";
 import { NAVIGATION_STATE } from "@constants/navigationState";
 import MetaTag from "@components/commons/meta/MetaTag";
 import { useHeader, useModal } from "@hooks";
+import { useLazyPostGuestBookingList } from "@apis/domains/bookings/queries";
 
 const NonMbLookup = () => {
   const navigate = useNavigate();
-
   const { openAlert } = useModal();
   const { setHeader } = useHeader();
+  const { fetchBookingList } = useLazyPostGuestBookingList();
 
-  // 하단 내역 확인 버튼 활성화/비활성화 상태
+  const [formData, setFormData] = useState({
+    bookerName: "",
+    birth: "",
+    number: "",
+    password: "",
+  });
   const [btnActive, setBtnActive] = useState(false);
-  // 하단 내역 확인 버튼 클릭 시 true
-  const [isReadyRequest, setIsReadyRequest] = useState(false);
-  // API 연결 후 들어오는 상태
-  const [dataState, setDataState] = useState(0);
-
-  const handleBtnOn = () => {
-    setBtnActive(true);
-  };
+  const [isFetching, setIsFetching] = useState(false);
 
-  const handleBtnOff = () => {
-    setBtnActive(false);
+  // 입력 값 변경 핸들러
+  const handleInputChange = (field: string, value: string) => {
+    setFormData((prev) => ({ ...prev, [field]: value }));
   };
 
-  const handleBtnState = () => {
-    setIsReadyRequest(true);
-  };
+  // 버튼 상태 활성화/비활성화
+  useEffect(() => {
+    const { bookerName, birth, number, password } = formData;
+    if (bookerName && birth.length === 6 && number.length === 13 && password.length === 4) {
+      setBtnActive(true);
+    } else {
+      setBtnActive(false);
+    }
+  }, [formData]);
 
-  const handleData = (status: number) => {
-    setDataState(status);
-  };
+  // 버튼 클릭 시 요청 실행
+  const handleBtnClick = async () => {
+    const { bookerName, birth, number, password } = formData;
 
-  const handleAlert = () => {
-    openAlert({
-      title: "일치하는 정보가 없습니다. \n 확인 후 다시 조회해 주세요",
-      okCallback: () => {
-        setIsReadyRequest(false);
-        setDataState(0);
-      },
-    });
-  };
+    try {
+      setIsFetching(true);
 
-  const handleLeftBtn = () => {
-    navigate("/main");
-  };
+      const bookingData = await fetchBookingList({
+        name: bookerName,
+        phone: number,
+        password,
+        birth,
+      });
 
-  useEffect(() => {
-    if (dataState === 404) {
-      handleAlert();
-    } else {
-      // API 붙이면 오류 맞춰서 처리하기
-      // console.log("오류 처리");
+      if (bookingData) {
+        navigate("/lookup", {
+          state: {
+            bookerName,
+            number,
+            password,
+            birth,
+          },
```

Comment:
p2: 여기서요! 

```suggestion
  const handleBtnClick = async () => {
    try {
      setIsFetching(true);

      const bookingData = await fetchBookingList(formData);

      if (bookingData) {
        navigate("/lookup", {
          state: formData,
```

이렇게 변수 네이밍만 조금 수정해줘도 더 쉽게 사용할 수 있어요!